package.json

```diff
@@ -7,6 +7,8 @@
     "postinstall": "(cd frontend && npm install --unsafe-perm) #Install npm packages in frontend also",
     "migrate": "./manage.py migrate",
     "migrate:integration": "./migrate-integration.sh",
+    "db:create": "createdb -p 5432 -O user teamware_db",
```

Comment:
The db name is defined in .env, is it possible to get it from there in the package scripts?

Reply:
I'd just put it in for testing but it's probably best to leave it out so I've removed it with the latest commit

Reply:
But I did find a tutorial on how to add .env file into package.json scripts though https://www.genui.com/resources/env-variables-json